scripts-available/CDB_AnalysisCheck.sql

```diff
@@ -0,0 +1,62 @@
+-- Read configuration parameter analysis_quota_factor, making it
+-- accessible to regular users (which don't have access to cdb_conf)
+CREATE OR REPLACE FUNCTION _CDB_GetConfAnalysisQuotaFactor()
+RETURNS float8 AS
+$$
+BEGIN
+  RETURN CDB_Conf_GetConf('analysis_quota_factor')::text::float8;
```

Comment:
Yes, I'm aware. We're using this as an easy way to be able to modify the default, for tests or so... We may think of something better in the future.